src/commonmark/en/content/developer/web-api.md

```diff
@@ -8682,15 +8682,63 @@ detailed information about each object on a users dashboard.
 
 <!--DHIS2-SECTION-ID:webapi_searching_dasboards-->
 
-When setting a dashboard it is convenient from a consumer point of view
+When a user is building a dashboard it is convenient
 to be able to search for various analytical resources using the
 */dashboards/q* resource. This resource lets you search for matches on
 the name property of the following objects: charts, maps, report tables,
```

Comment:
I would use the word "type" consistently. "This resource lets you search for matches on the name property of the following object **types**: ..."